war_or_peace_runner.rb

```diff
@@ -0,0 +1,2 @@
+require './lib/game'
+start_game = Game.new.start
```

Comment:
This runner looks great. Although your `Game` class doesn't appear to be done just yet, I like the decision you made to keep all the game setup pieces (like creating the decks) in that class so that all your runner needs to be responsible for is the act of starting the game.